elpy-shell.el

```diff
@@ -269,7 +269,7 @@ Python process. This allows the process to start up."
          (proc (get-buffer-process bufname)))
     (if proc
         proc
-      (unless (executable-find python-shell-interpreter)
+      (unless (executable-find python-shell-interpreter t)
```

Comment:
Can you make this into a variable with a default of false?
Thank You!

Reply:
why would it ever need to be `nil`?
is the file is local, the check is local either way.
is the file is remote, the local check (with `nil`) is _pointless_ because the actual invocation of `python-shell-interpreter` will be done _remotely_ anyway.
thus using `nil` for the `executable-find` call here is always wrong.